src/modules/organisation/api/organisation.uc.spec.ts

```diff
@@ -83,4 +87,51 @@ describe('OrganisationUc', () => {
             );
         });
     });
+
+    describe('findAll', () => {
+        const findOrganisationDto: FindOrganisationDto = {
+            kennung: 'kennung',
+            name: 'name',
+            typ: OrganisationsTyp.SCHULE,
+            offset: 0,
+            limit: 0,
+        };
+
+        it('should find all organizations that match with query param', async () => {
+            const firstOrganisation: OrganisationDo<true> = DoFactory.createOrganisation(true);
+            const secondOrganisation: OrganisationDo<true> = DoFactory.createOrganisation(true);
```

Comment:
If EW-524 is merged, DoFactory contains a createMany method.

Reply:
✅